api/src/http.rs

```diff
@@ -413,6 +415,17 @@ impl Default for ProxyConfig {
     }
 }
 
+/// Configuration information for mirror.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct MirrorConfig {
+    /// Dragonfaly http scheme, either 'http' or 'https'.
+    pub scheme: String,
```

Comment:
Can use mirror registry or pull-through cache rather than Dragonfly? In addition, I think docker registry can also work together with nydusd with the mirror configuration

Reply:
Fixed.